### PR TITLE
Automake updates

### PR DESCRIPTION
**Pull request by mayank18049**

### CODE REVIEW

File: python/Makefile.am [hunk 1]
```diff
@@ -7,16 +7,13 @@ if PYTHON
 
 noinst_SCRIPTS = python-build
 
-$(srcdir)/dnet.c: $(srcdir)/dnet.pyx
-	pyrexc $(srcdir)/dnet.pyx
```
Comment (lnslbrty) [inline python/Makefile.am:11]: Isn't it required for generating dnet.c via Makefile?

Comment (mayank18049) [reply python/Makefile.am:11]: We will not be making any changes to the created dnet.c so It does not make sense to specifically generate it from Makefile before a build or install.

**Comment by ofalk** [comment]
@mayank18049 This PR is currently conflicting with at configure.in - is this still valid? If so, please adapt accordingly. Thx!

**Comment by mayank18049** [comment]
@ofalk I just changed the author name in configure.in not much.

**Comment by mayank18049** [comment]
@ofalk  I think this should work

**Pull request merged**